components/content-service-api/initializer.proto

```diff
@@ -14,6 +14,7 @@ message WorkspaceInitializer {
         CompositeInitializer composite = 5;
         FileDownloadInitializer download = 6;
         FromBackupInitializer backup = 7;
+        FromSnapshotVolumeInitializer snapshot_volume = 8;
```

Comment:
If we rename it as `FromVolumeSnapshotInitializer`, would it better?
Since it aligns to the naming https://github.com/kubernetes-csi/external-snapshotter/tree/master/client/config/crd, WDYT?

Reply:
You are right, especially that DB entity is correctly named VolumeSnapshot. I will fix.

Reply:
@jenting fixed! PTAL